### PR TITLE
Script updated to support YugaByte version 1.3.0

### PR DESCRIPTION
**Pull request by vp15591**
Script updated to support YugaByte version 1.3.0 and also fixed the  [issue #1952](https://github.com/YugaByte/yugabyte-db/issues/1952)

### CODE REVIEW

File: scripts/install_software.sh [hunk 1]
```diff
@@ -14,10 +14,12 @@ if [[ $# -eq 3 ]]; then
    YB_DOWNLOAD_LOCATION=$3
```
Comment (ramkumarvs) [inline scripts/install_software.sh]: I don't think we should have editions anymore.. What if the user wants to deploy a different version other than 1.3.0.0. In the current approach that would not work. 

Also why are we using standalone scripts, we should use the common utilities repo.

File: yugabyte_cloudformation.yaml [hunk 2]
```diff
@@ -36,7 +36,7 @@ Mappings:
       CIDR: "10.0.2.0/24"
   YugaByteConfig:
     Edition: 
-      Value: "ce"
+      Value: "YB"
```
Comment (ramkumarvs) [inline yugabyte_cloudformation.yaml]: We should get rid of edition all together.

File: scripts/install_software.sh (hunk 1 above)
Comment (vp15591) [reply scripts/install_software.sh]: Cloudformation script is different than the utility repo script that's why we are using the standalone scripts as of now. In a future release, we will try to use common utility repo.

File: yugabyte_cloudformation.yaml (hunk 2 above)
Comment (vp15591) [reply yugabyte_cloudformation.yaml]: removed edition dependency.

**Pull request merged**